### DISCUSSION TITLE
Unable to run tutorial 1.2 Hello Quad

### DISCUSSION THREAD

**Discussion by erikmartinessanches** [dotnet/Silk.NET#381] [Q&A]
# Summary
Unable to run tutorial 1.2 Hello Quad. (The only project I've tried.)

# Steps to reproduce
- Platform: Laptop, windows 10 on Macbook Pro retina, 13 inch, mid 2014, core i5.
- Framework Version: .NET Core 5 (I also have .NET Core 3.1.404 installed.)
- API: OpenGL 4.0, (intel iris 5100)
1. Opened the project and restored dotnet packages.
2. Selected "Tutorial 1.2 - Hello Quad.csproj" from the run buttom. 
3. Started without debugging.

Error message (translated):
Unhandled exception: System.IO.FileNotFoundException: Unable to read the file or the assembly System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a or one of its dependencies. Unable to find the file.

C:\Users\path\Silk.NET\examples\CSharp\Tutorial 1.2 - Hello quad\bin\Debug\netcoreapp3.1\Tutorial.dll (process 7740) exited with code -532462766.

**Comment by Perksey** [comment]
This doesn't seem at first to be a Silk.NET bug seeing as we don't knowingly reference System.Runtime anywhere. Can you do a `dotnet new console` and run that?

**Comment by Perksey** [comment]
Converting to a discussion while we investigate. If it's an actual bug we can open a new issue.
